man/set_mass_vaccination.Rd

```diff
@@ -1,23 +1,27 @@
 % Generated by roxygen2: do not edit by hand
 % Please edit documentation in R/vaccine_parameters.R
-\name{set_mass_rtss}
-\alias{set_mass_rtss}
-\title{Parameterise an RTS,S mass distribution strategy}
+\name{set_mass_vaccination}
+\alias{set_mass_vaccination}
+\title{Parameterise a vaccine mass distribution strategy}
 \usage{
-set_mass_rtss(
+set_mass_vaccination(
   parameters,
+  profile,
   timesteps,
   coverages,
   min_ages,
   max_ages,
   min_wait,
```

Comment:
The min_wait for mass vaccination exists on dev. Could we have a separate issue for that if we really want to remove it? Is there not a scenario where someone wants to model epi and mass vaccination in the same run?

Reply:
There is the possibility - I was thinking this would always be set through set_vaccine_epi, but you're right that both functions would need it depending on the order of vaccination.